modules/serverless-create-kn-trigger.adoc

```diff
@@ -1,8 +1,10 @@
+// Module included in the following assemblies:
+//
+// * /serverless/event_workflows/serverless-using-brokers.adoc
+
 [id="serverless-create-kn-trigger_{context}"]
 = Creating a trigger using `kn`
 
-The Knative CLI provides a set of `kn trigger` commands that can be used to create and manage triggers.
```

Comment:
Is there a reason that you moved this introductory text? It's typically something that we'd want to include in a module.

Reply:
I'm actually not sure, may have been a mistake. Re-added, thank you